test/elixir/README.md

```diff
@@ -43,8 +43,9 @@ X means done, - means partially
   - [X] Port cookie_auth.js
   - [X] Port copy_doc.js
   - [ ] Port design_docs.js
-  - [ ] Port design_options.js
-  - [ ] Port design_paths.js
+  - [X] Port design_docs_query.js
```

Comment:
Just to be clear, `design_docs.js` still hasn't been ported, am I right? So you moved `design_docs_query.js`-related tests into their own file because earlier they were in the wrong place?

Reply:
Thanks @dottorblaster 
design_docs.js is not ported, i'm working on it just know. 
A couple of tests existed in the file design_docs_test.exs but it was not the correct place. I've moved those test to design_docs_query_test.exs.

Reply:
Totally legit, thanks for noticing it!